pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -482,6 +485,15 @@ public void updateAll() {
     // there by each broker via updateLocalBrokerData.
     private void updateAllBrokerData() {
         final Set<String> activeBrokers = getAvailableBrokers();
+        Collection<String> newBrokers = CollectionUtils.subtract(activeBrokers, knownBrokers);
```

Comment:
I think it's a little weird to put a cleanup in `updateAllBrokerData`. I suggest we could put these logic in a new method like `cleanup()`, and call the method from `updateAll`.

Reply:
@Jason918  done.